contents/docs/product-analytics/person-properties.mdx

```diff
@@ -105,6 +105,10 @@ posthog.capture("distinct_id", "event_name", new HashMap<String, Object>() {
 
 </MultiLanguage>
 
+## Limitations
+
+Person properties have a size limit of **512KB** per person record. If you attempt to create or update a person with properties that exceed this limit, the update will be rejected and an ingestion warning will be logged. This is a limit on the combined size of all properties on the person record, not individual property values.
```

Comment:
I think this might work better as a callout box
```suggestion
<CalloutBox icon="IconWarning" title="Size limitation" type="caution">

Person properties have a size limit of **512KB** per person record. If you attempt to create or update a person with properties that exceed this limit, the update will be rejected and an ingestion warning will be logged. This is a limit on the combined size of all properties on the person record, not individual property values.

</CalloutBox>
```